### ISSUE TITLE
Assembly Process

### ISSUE THREAD

**Issue by AbeEstrada** [MicrosoftEdge/magic-mirror-demo#25]
The [Assembly Process](https://github.com/MicrosoftEdge/magic-mirror-demo/blob/master/.github/ASSEMBLY.md) needs more details.

- Step 4: Requires an image to know how is the monitor going to rest on those legs.
- Step 5: Where do I put it? How about the connections?
- Step 7: How it is supposed to look when finished? An image will help a lot.

**Comment by wardellchandler** [comment]
To add to this, I also don't see anything that says the dimensions of the 2 way glass.

**Comment by AbeEstrada** [comment]
@wardellchandler once you have everything installed or maybe just the cabinet you can measure how big you want the 2 way glass

**Comment by andypavia** [comment]
@AbeEstrada  Thanks for opening the issue! 

> Step 4: Requires an image to know how is the monitor going to rest on those legs.
I just added an image to step 4 to better illustrate how the monitor should rest on the legs. 

> Step 5: Where do I put it? How about the connections?
We just placed the Pi at the bottom of the cabinet. 

> Step 7: How it is supposed to look when finished? An image will help a lot.
Let me take an image of the finished product. I'll upload it tomorrow.

@wardellchandler below are the dimensions of the sheet. I added them to the assembly instructions.
2-Way Mirrored Acrylic Sheets
Thickness: 1/8"
Width: 13”
Length: 23 5/8”
